.changeset/smart-schools-know.md

```diff
@@ -0,0 +1,5 @@
+---
+'@redux-saga/delay-p': patch
+---
+
+prevent delay from containing a delay time that exceeds the maximum value
```

Comment:
```suggestion
Fixed an issue with arguments that exceed the maximum value for the internally-used `setTimeout`. Previously it could overflow based on the input that was too big and thus a timeout could resolve immediately.
```